gematria/datasets/llvm-exegesis_warpper.S

```diff
@@ -0,0 +1,32 @@
+# LLVM-EXEGESIS-DEFREG RAX 10000
```

Comment:
Having this produced/embedded in the executable seems like it would be cleaner than loading it in at runtime? Not sure what @ondrasej thinks about that though.